packages/client/src/components/Footer/Footer.jsx

```diff
@@ -1,6 +1,27 @@
 import { Box, Container, Divider, Grid, Typography } from '@mui/material';
 import { useTheme } from '@mui/material/styles';
 import React from 'react';
+import styled from '@emotion/styled';
+
+const StyledBox = styled(Box)`
+  margin-top: ${(props) => props.theme.spacing(8)};
+  padding: ${(props) => props.theme.spacing(0, 0)};
+  width: 100%;
+  bottom: 0;
+`;
+
+const StyledTypography = styled(Typography)`
+  color: #ffffff;
+  font-family: Inter;
+  font-size: 1.5rem;
+  line-height: 1.813rem;
```

Comment:
Same remark as Babak mentioned in the #87.
Otherwise all good! 👍🏽

Reply:
doing it! 🫡